cypress.json

```diff
@@ -7,5 +7,7 @@
     "runMode": 2,
     "openMode": 0
   },
-  "blockHosts": "*.google-analytics.com"
+  "blockHosts": "*.google-analytics.com",
+  "testFiles": ["**/**/*.spec.ts"],
+  "experimentalNetworkStubbing": true
```

Comment:
Allow us to try out `route2` "stubbing" which has become their new intercept tool (called `.intercept`).

Reply:
Given that we do want to upgrade cypress in the very near future, do you think it is worthwhile starting to use a configuration option that is removed in [version 6](https://docs.cypress.io/guides/references/changelog#6-0-0), which enables the use of a feature that is also removed in [version 7](https://docs.cypress.io/guides/references/changelog#7-0-0)?

Reply:
No I don't. Truthfully, I think the best path forward would be to wait to add this test until we update. I added this as a work around just in case folks felt strongly that the ticket for site search test coverage be closed before the end of the year.

Reply:
I assume you don't need this for this changeset anymore?